frontend/src/metabase/query_builder/components/filters/pickers/RelativeDatePicker.styled.tsx

```diff
@@ -0,0 +1,72 @@
+import { color, alpha } from "metabase/lib/colors";
```

Comment:
We prefer creating a separate folder for a component and its styles, utils, tests, inner components, etc

Reply:
Created a `pickers/DatePicker` folder